rust-symcrypt/README.md

```diff
@@ -4,36 +4,19 @@ This crate provides friendly and idiomatic Rust wrappers over [SymCrypt](https:/
 
-**Note:** As of version `0.2.0`, only `Windows AMD64`, and [`Azure Linux`](https://github.com/microsoft/azurelinux) are fully supported, with partial support for other Linux distros such as `Ubuntu`.
+**`symcrypt` version `0.3.0` is based off of `SymCrypt v103.5.0.1`.**
 
-## Quick Start Guide
-
-### Windows:
-Download the `symcrypt.dll` and `symcrypt.lib` for you corresponding CPU architecture from the [SymCrypt Releases Page](https://github.com/microsoft/SymCrypt/releases/tag/v103.4.2) and place them somewhere accessible on your machine.
-
-Set the required `SYMCRYPT_LIB_PATH` environment variable. You can do this by using the following command:
-
-`setx SYMCRYPT_LIB_PATH "<your-path-to-symcrypt-lib-folder>"`
-
-During runtime, Windows will handle finding all needed `dll`'s in order to run the intended program, this includes our `symcrypt.dll` file.
-
-Here are 2 recommended options to ensure your `symcrypt.dll` is found by Windows during runtime, for more info please see [Dynamic-link library search order](https://learn.microsoft.com/en-us/windows/win32/dlls/dynamic-link-library-search-order).
+To view a detailed list of changes please see the [releases page](https://github.com/microsoft/rust-symcrypt/releases/).
 
-1. Put the `symcrypt.dll` in the same folder as your output `.exe` file. If you are doing development (not release), the common path will be: `C:\your-project\target\debug\`.
-2. Permanently add the `symcrypt.dll` path into your System PATH environment variable. Doing this will ensure that any project that uses the SymCrypt crate will be able to access `symcrypt.lib`
 
-For more information please see the `INSTALL.md` file on the [`rust-symcrypt`](https://github.com/microsoft/rust-symcrypt/tree/main/rust-symcrypt) page
+### Supported Configurations
 
-### Linux:
-Download and all of the `libsymcrypt.so*` files for you corresponding CPU architecture from the [SymCrypt Releases Page](https://github.com/microsoft/SymCrypt/releases/tag/v103.4.2).
-
-Support for `Debian` and `Ubuntu` via package manager is in the works, for now you must place the `libsymcrypt.so*` files into linker load path. The way that this is set will vary between distros. On most distros it set via the environment variable `$LD_LIBRARY_PATH`. 
-
-Package manager support is already available if you are using `Azure Linux` via `tdnf install symcrypt`.
-
-For more information please see the `INSTALL.md` file on the [`rust-symcrypt`](https://github.com/microsoft/rust-symcrypt/tree/main/rust-symcrypt) page
-
-**Note:** This path may be different depending on your flavour of Linux. The goal is to place the `libsymcrypt.so*` files in a location where the your Linux distro can find the required libs at build/run time.
+| Operating Environment | Architecture      | Dynamic Linking |
+| --------------------- | ----------------- | ----------- |
+| Windows user mode     | AMD64, ARM64      | ✅          | 
+| Ubuntu (Tested via WSL)       | AMD64, ARM64      | ✅          | 

```

Comment:
>(Tested via WSL) [](http://example.com/codeflow?start=9&length=16)

Don't think it's necessary to mention that this was tested with WSL. WSL 2 is basically just a Linux VM so there should be no difference.